src/pages/Strategies/components/StrategyForm/StrategyDepositForm.tsx

```diff
@@ -51,30 +67,61 @@ const StrategyDepositForm = ({ strategy, position }: StrategyDepositFormProps):
   );
 
   const handleSubmit = (values: StrategyDepositFormData) => {
-    const transactionData = getTransactionArgs(values);
-
-    if (!transactionData) return;
-
-    const { monetaryAmount } = transactionData;
-
-    transaction.execute(WRAPPED_TOKEN, monetaryAmount);
+    if (proxyAccount) {
+      const depositTransactionData = getDepositTransactionArgs(values);
+
+      if (!depositTransactionData) return;
+
+      const { monetaryAmount } = depositTransactionData;
```

Comment:
As in the other forms, you need to deduct the transaction fee from the amount in case the selected fee token matches the amount currency